inactivity.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: Maintainer and Repository Inactivity
+nav_order: 6
+---
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# Inactivity Policies
+
+As part of the normal lifecycle of a project codebases and maintainers come and go. This document formalizes a Hyperledger wide policy for archiving projects and moving maintainers to emeritus status when a project does not have an explicit policy of their own.
+
+## Maintainer Inactivity
+
+Hyperledger very much appreciates the contributions of all maintainers but removing write privileges is in the interest of an orderly and secure project.
+
+Maintainers who have not contributed to a Hyperledger project for (`exact timeframe to be decided`) 3 / 4 / 6 months will be removed from the relevant GitHub teams.
+
+Activity can be code contributions, code reviews, issue reporting, or any other such activity trackable by GitHub attributed to a Hyperledger repository.
+
+Maintainers will be notified of this via an "at" @ mention in a pull request moving their name from a projects active contributors to the emeritus contributors.
+
+Maintainers who express a desire to continue contributing will, at the maintainers request, be given a 3-month extension to contribute to the project.
```

Comment:
Not sure if the second maintainer in this statement refers to the active maintainers or the identified inactive maintainer. It would be good to clarify that the identified inactive maintainer is the one making the request. Do the active maintainers have the ability to reject this request?

Reply:
I will workshop the phrasing. The intent is that the maintainer who is proposed to be removed can make the request, and they get a single three month extension.  If they need more than 9 months my view off they should go emeritus and when they are ready to return re-ask for active status with the current maintainers.

Reply:
Agree with a single extension. Just want that to be obvious so there are no chances of someone who just keeps asking for extensions and saying that it is okay based on the policy.